### PR TITLE
End to end tests for DoNotEndAppStart class

### PR DESCRIPTION
**Pull request by SmartbearYing**
## Testing

End to end tests for a splash activity which is annotated as DoesNotEndAppStart and ensure that the end time of the AppStart is equal to the end of ViewLoad from a main activity.

### CODE REVIEW

**Review by lemnik — APPROVED** [review]
LGTM

File: features/instrumentation.feature [hunk 1]
```diff
@@ -89,99 +89,158 @@ Feature: Automatic creation of spans
...
+      | attribute             | type        | value           |
+      | bugsnag.span.category | stringValue | view_load_phase |
+      | bugsnag.phase         | stringValue | ActivityStart   |
+      | bugsnag.view.name     | stringValue | MainActivity    |
+
+    * a span named "[ViewLoadPhase/ActivityResume]MainActivity" contains the attributes:
+      | attribute             | type        | value           |
+      | bugsnag.span.category | stringValue | view_load_phase |
+      | bugsnag.phase         | stringValue | ActivityResume  |
+      | bugsnag.view.name     | stringValue | MainActivity    |
```
Comment (twometresteve) [inline features/instrumentation.feature:245]: If I'm reading this right, these are two sets of the same 3 spans?  Could they potentially match on just three unique spans and there be another 3 spans of junk present?  Did you intend for one half of them to cover the new activity?

**Review by twometresteve — APPROVED** [review]
LGTM - just the one question.

File: features/instrumentation.feature (hunk 1 above)
Comment (lemnik) [reply features/instrumentation.feature:245]: Oops, nicely spotted. That second set should be the new SplashScreenActivity - will fix.

**Pull request merged**